aptos-move/framework/aptos-experimental/sources/trading/market/market.move

```diff
@@ -105,13 +105,17 @@ module aptos_experimental::market {
         market: address,
         // TODO: remove sequential order id generation
         last_order_id: u64,
+        // Incremental fill id for matched orders
+        next_fill_id: u64,
         config: MarketConfig,
         order_book: OrderBook<M>
     }
 
     struct MarketConfig has store {
         /// Weather to allow self matching orders
-        allow_self_trade: bool
+        allow_self_trade: bool,
+        /// Whether to block the sending of all events for the market
+        halt_events: bool
```

Comment:
I would prefer to name this something like `allow_events_emission` and also flip the functionality of the flag.